tools/py/Instanton_postproc.py

```diff
@@ -511,7 +511,7 @@ def get_rp_freq(w0, nbeads, temp, mode="rate"):
             d_min[i] = float(aux[i])
         d_min = d_min.reshape((natoms * 3))
         out.close()
-        ww = get_rp_freq(d_min, nbeads, temp, mode="splitting")
+        ww = get_rp_freq(d_min**2, nbeads, temp, mode="splitting")
```

Comment:
I think that with this change, the check in line 202 
`    if np.amin(w0) < 0.0:
        print("@get_rp_freq: We have a negative frequency, something is going wrong.")
        sys.exit()`
        
        will never be triggered. I guess this check can be made when reading the file here instead?